NEWS.md

```diff
@@ -19,6 +19,7 @@
 
 ### Bug fixes
  * `matrix_form(lsting, TRUE)` no longer throws an error.
+ * Fixed pagination tests to have compatibility with non-`UTF8` coding by specifying `hsep = "-"`.
```

Comment:
This doesn't warrant a NEWS item, please remove.

Reply:
I think this is quite a definition of the section called "Bug fixes", but it will be anyway squashed in the future